WebCryptoAPI/util/helpers.js

```diff
@@ -130,6 +130,10 @@ function assert_goodCryptoKey(key, algorithm, extractable, usages, kind) {
         assert_equals(key.algorithm.hash.name.toUpperCase(), algorithm.hash.toUpperCase(), "Correct hash function");
     }
 
+    if (/^(?:Ed|X)(?:25519|448)$/.test(key.algorithm.name)) {
+        assert_false('namedCurve' in key.algorithm, "Does not have a namedCurve property")
```

Comment:
This is super nitpicky but since the surrounding code uses semicolons, I would do so here too
```suggestion
        assert_false('namedCurve' in key.algorithm, "Does not have a namedCurve property");
```